dotCMS/src/main/java/com/dotcms/util/SecurityUtils.java

```diff
@@ -8,7 +8,6 @@
 
 import com.dotmarketing.beans.Host;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.dotmarketing.beans.Host'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)